deployment/main.tf

```diff
@@ -22,7 +22,7 @@ module "vpc" {
 
 module "common-fate-deployment" {
   source  = "common-fate/common-fate-deployment/aws"
-  version = "1.45.2"
+  version = "2.27.1"
```

Comment:
This should be `2.8.1`: https://github.com/common-fate/terraform-aws-common-fate-deployment/releases/tag/v2.8.1